MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToggleButton.xaml

```diff
@@ -136,7 +136,7 @@
 						</VisualStateManager.VisualStateGroups>
 						<Ellipse Fill="{TemplateBinding Background}" Width="{TemplateBinding Width}" Height="{TemplateBinding Height}" x:Name="UncheckedEllipse" Opacity=".54" />
 						<Ellipse Fill="{TemplateBinding Background}" Width="{TemplateBinding Width}" Height="{TemplateBinding Height}" x:Name="CheckedEllipse" />
-						<Viewbox Width="24" Height="24" x:Name="CheckMark" Visibility="Hidden" HorizontalAlignment="Center" VerticalAlignment="Center">
+						<Viewbox Width="24" Height="24" x:Name="CheckMark" Visibility="Hidden" HorizontalAlignment="Center" VerticalAlignment="Center" FlowDirection="LeftToRight">
```

Comment:
Exactly the same as Icons and checkboxes, The Checked mark inside toggle button should always look the same